puppet/modules/web/manifests/init.pp

```diff
@@ -137,16 +137,16 @@
     {
       path            => $docs_directory,
       options         => ['Indexes', 'FollowSymLinks', 'MultiViews'],
```

Comment:
Please correct the alignment here (and in the attrs) to avoid the empty whitespace.

Reply:
Done.